builder/edition_utilities.livecodescript

```diff
@@ -0,0 +1,58 @@
+﻿script "EditionUtilities"
+constant kEditionNames="community,communityplus,indy,business,commercial"
+function editionNames
+   return kEditionNames
+end editionNames
+
+function editionTitleCase pEdition
+	switch pEdition
+	case "community"
+		return "Community"
+	case "communityplus"
+		return "CommunityPlus"
+	case "indy"
+		return "Indy"
+	case "business"
+		return "Business"
+	case "commercial"
+		return "Commercial"
```

Comment:
@livecodeian is there such a thing as the `Commercial` edition? This could be a little confusing with `the editionType` of Indy = `commercial`  but `editionCompare("indy","commercial") = -2`

Reply:
The "Commercial" edition is only relevant to server, which only differentiates between "Commercial" and "Community"

Reply:
ah I see